### PR TITLE
Storm scene: miscellaneous popover componentizations

### PR DESCRIPTION
**Pull request by modernserf**
## Links
* Remix link https://storm-scene.glitch.me/

## Changes:
* componentize Delete Project popover
* use Popover components on search form & homepage video
* remove unused (?) sign-in page?

## How To Test:
* visit https://storm-scene.glitch.me/
* verify that the "how it works" video opens and closes if it loses focus
* verify that the search autocomplete opens when you type in the search box and closes when you focus elsewhere
* log in
* create a project
* delete that project from its project page
* verify that the delete project pop works

### CODE REVIEW

**Comment by taravancil** [comment]
Might want to check with @oliviacpu to see if she needs the sign in page for the auth work she's doing

File: src/presenters/overlays/overlay-video.js [hunk 1]
```diff
@@ -33,9 +32,9 @@ const OverlayVideo = ({ children }) => {
   const track = useTracker('How it works clicked');
   return (
     <PopoverContainer>
-      {({ visible, setVisible }) => {
```
Comment (taravancil) [inline src/presenters/overlays/overlay-video.js]: Something funny going on here. If you close the video with the escape button, it toggles repeatedly between open and closed

**Comment by taravancil** [comment]
> Might want to check with @oliviacpu to see if she needs the sign in page

nvm, that's def not being used anywhere

File: src/presenters/overlays/overlay-video.js (hunk 1 above)
Comment (modernserf) [reply src/presenters/overlays/overlay-video.js]: updated.

**Pull request merged**